submissions/LuckyDnepr/Hooli-popup-task/index.html

```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles-menu.css">
+    <title>Hooli menu</title>
+</head>
+
+<body>
+    <header class="header">
+        <input id="nav_keeper_input" type="checkbox" tabindex="1">
+        <div class="nav_keeper_lines">
+            <span class="nav_keeper_line line1"></span>
+            <span class="nav_keeper_line line2"></span>
+            <span class="nav_keeper_line line3"></span>
+        </div>
+        <nav class="nav">
+            <input id="menu_show_more_input" type="checkbox" tabindex="10">
+            <label id="menu_show_more_label" for="menu_show_more_input">
+                <span class="menu_show_more show_more_span">Show more</span>
+                <span class="menu_show_more show_less_span">Show less</span>
+            </label>
+            <ul class="nav_menu" id="nav_menu" >
+                <li class="nav_menu_item">
+                    <a href="#Applications" class="nav_menu_item_link" tabindex="2">
+                        <div class="nav_menu_item_link_container">
+                            <img class="menu_icons" src="./icons/apps.png" title="Apps">
+                            <p>Applications</p>
+                        </div>
+                    </a>
+                </li>
+                <li class="nav_menu_item">
+                    <a href="#Blogs" class="nav_menu_item_link" tabindex="3">
```

Comment:
no need to explicitly specify the value of `tabindex`.
`a` tag support keyboard interaction because `a` tag is focusable.

Focusable elements are `links, input fields, and buttons. `